operators/operator-reference.adoc

```diff
@@ -6,6 +6,13 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 include::modules/cloud-credential-operator.adoc[leveloffset=+1]
+[discrete]
+[id="red-hat-operators-cco-addtl-resources"]
+=== Additional resources
```

Comment:
Might want to link to _Rotating or removing cloud provider credentials_ section as well